api/application.go

```diff
@@ -299,11 +319,13 @@ func (a *ApplicationAPI) UploadApplicationImage(ctx *gin.Context) {
 				os.Remove(a.ImageDir + app.Image)
 			}
 
-			app.Image = name + ext
-			a.DB.UpdateApplication(app)
+			app.Image = name
```

Comment:
Now the image contains the image dir, like this it can't be resolved anymore. Example URL: `http://localhost:8080/image/data/images/LZZCuJ5E5nb_5qGgDo_lwqEne.PNG`